Flapjack/CoreData/CoreDataAccess.swift

```diff
@@ -120,15 +120,12 @@ public final class CoreDataAccess: DataAccess {
                 return
             }
 
-            self.addDefaultPersistentStores(async: asynchronously) { [weak self] fatalError in
-                guard let self = self, fatalError == nil else {
+            self.addDefaultPersistentStores(async: asynchronously) { fatalError in
```

Comment:
I've started at this file for a while now and can't figure out what this change does differently...

Reply:
I'm setting `self.container.viewContext.automaticallyMergesChangesFromParent` and `self.isStackReady` inside the end of `addDefaultPersistentStores` in a `finally` block, instead of doing it here. This was necessary so that we could use the same settings at the tail end of two events:

- setting up the initial stack, which is where the settings were set before, and
- rebuilding the stack after a delete

Reply:
In the second case, is that something that would happen outside of Flapjack?

Reply:
I mean it would get _triggered_ from outside of Flapjack, but Flapjack was the one responsible for setting these values when it setup the stack the first time, so it should be responsible for setting them up each subsequent time as well.

Reply:
And rebuilding the stack happens inside of Flapjack, I should say — all the code for it is already there (and we're already using it)

Reply:
Ah.. ok. Missed the calls to `addEfaultPersistentStores` in `deleteDatabase`.  So, migration only needs to happen once,  I'm guessing.

Reply:
Yeah, migrations become totally moot when you're nuking the old database files.